src/renderer/services/db/api/repositories/validator/attestations.ts

```diff
@@ -48,6 +48,6 @@ export class ValidatorAttestationsRepository extends BulkRepository<Attestation>
 
     private getAttestationKey(pubKey: BLSPubkey, attestation: Attestation): Buffer {
         const epoch = intToBytes(attestation.data.target.epoch, 2);
-        return Buffer.concat([pubKey, epoch, attestation.signature]);
+        return Buffer.concat([pubKey.valueOf() as Uint8Array, epoch, attestation.signature.valueOf() as Uint8Array]);
```

Comment:
What's with this return type of `valueOf` ?

Reply:
Underlying implementation is binary tree so valueOf() allows us to proxy tree and return it as Uint8Array. If it were structural backing, underlying representation is Uint8Array so valueOf() on Uint8Array is still Uint8Array.
You can read about that here: https://github.com/chainsafe/ssz#backings